jobs/virtcontainers-fedora-26-PR/config.xml

```diff
@@ -92,6 +92,8 @@ set -e
 
 export ghprbPullId
 export ghprbTargetBranch
+export ghprbAuthorRepoGitUrl
```

Comment:
Can you add a comment in the commit message that these changes were to comply with the virtcontainers CI changes? 
https://github.com/containers/virtcontainers/commit/1a638c0f173b9aa06d8e03943ae604655f1719b8
Or better, add a new commit.

Reply:
@chavafg , added new commit !